cmd/find-main.go

```diff
@@ -0,0 +1,182 @@
+/*
+ * Minio Client (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package cmd stores all of the mc utilities
+package cmd
+
+import (
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	"github.com/minio/mc/pkg/console"
+)
+
+// find specific flags
+var (
+	findFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "Find files of a matching name",
+		},
+		cli.StringFlag{
+			Name:  "path",
+			Usage: "Find global matches on the entire path",
+		},
+		cli.StringFlag{
+			Name:  "regex",
+			Usage: "Matches all files with PCRE regular expression",
+		},
+		cli.StringFlag{
+			Name:  "print",
+			Usage: "Prints the found files given args to STDOUT",
+		},
+		cli.StringFlag{
+			Name:  "exec",
+			Usage: "Spawns a process by executing ARGS per-file found",
+		},
+		cli.StringFlag{
+			Name:  "ignore",
+			Usage: "Ignore the specified pattern",
+		},
+		cli.StringFlag{
+			Name:  "newer",
+			Usage: "Return all time values which fit within the given params",
+		},
+		cli.StringFlag{
+			Name:  "older",
+			Usage: "Return all time values which fit within the given params",
+		},
+		cli.StringFlag{
+			Name:  "larger",
+			Usage: "Return all objects which fit within the confines of the given size param",
+		},
+		cli.StringFlag{
+			Name:  "smaller",
+			Usage: "Return all objects which fit within the confines of the given size params",
+		},
+		cli.StringFlag{
+			Name:  "maxdepth",
+			Usage: "Return all objects up to that depth",
+		},
+		cli.BoolFlag{
+			Name:  "watch",
+			Usage: "Prefroms given args every time that an object is created",
```

Comment:
`Prefroms` -> `Performs` - there are multiple different misspellings of the same word in many places in this patch.